src/components/stepforms/widgets/Autocomplete.vue

```diff
@@ -7,9 +7,11 @@
       :variable-delimiters="variableDelimiters"
       :has-arrow="true"
       @input="updateValue"
+      @choosing="chooseVariable"
     >
       <multiselect
         :value="value"
+        :disabled="isChoosingVariable"
```

Comment:
That's a clever idea to avoid having problematic interactions while choosing.
However, it has several drawbacks:
- the internal state of the VariableInput needs to be synchronized with this upper-state variable (which is a shame, this state should only exists once somewhere, and not twice)
- this behavior will need to be adapted for each component we apply the VariableInput on, which is repetitive and diminish the interest of the VariableInput component itself.

Would you like to discuss a bit how we could solve this directly in the VariableInput component once and for all?